apps/web/src/components/ConnectedWalletDropdown.tsx

```diff
@@ -0,0 +1,29 @@
+import { twMerge } from "tailwind-merge";
+import { Wallet } from "./Icons";
+interface ConnectedWalletDropdownProps {
+  currentWalletAddress: string;
+}
+
+export default function ConnectedWalletDropdown(props: ConnectedWalletDropdownProps) {
+  const { currentWalletAddress } = props;
+
+  const displayedCurrentWalletAddress = `${currentWalletAddress.slice(0, 5)}...${currentWalletAddress.slice(-6)}`;
+
+  const wrapperPseudoElementClassName = "after:content-['▼'] after:absolute after:right-0 after:pointer-events-none";
```

Comment:
can just live `className`, much cleaner

Reply:
Added this because you'd suggested splitting things out to prevent line length growing too large just as an FYI!